src/electron/build.action.mjs

```diff
@@ -13,30 +13,30 @@
 // limitations under the License.
 
 import {getElectronBuildParameters} from './get_electron_build_parameters.mjs';
-import {getVersion} from '../build/get_version.mjs';
 import {runAction} from '../build/run_action.mjs';
 import electron, {Platform} from 'electron-builder';
 import copydir from 'copy-dir';
 import fs from 'fs/promises';
 import url from 'url';
 import {getRootDir} from '../build/get_root_dir.mjs';
 import path from 'path';
+import {getBuildEnvironment} from '../build/get_build_environment.mjs';
 
 const ELECTRON_BUILD_DIR = 'build';
 
 export async function main(...parameters) {
   const {platform, buildMode, stagingPercentage, publish} = getElectronBuildParameters(parameters);
```

Comment:
Get the `stagingPercentage` from `parameters`, not `getElectronBuildParameters`.

`getElectronBuildParameters` should not have to know about `stagingPercentage`